internal/controller/contact.go

```diff
@@ -148,7 +148,7 @@ func (c *contactController) UpdateContact(ctx *gin.Context) {
 // @Failure 400 {object}      util.HTTPError
 // @Failure 404 {object}      util.HTTPError
 // @Failure 500 {object}      util.HTTPError
-// @Router  /api/contacts/{id} [delete]
```

Comment:
Why removed `api` prefix?

Reply:
Because base path is documented in main.go